recipe/meta.yaml

```diff
@@ -20,24 +20,24 @@ requirements:
     - python
   run:
     - ansimarkup >=1.0.0
-    - click >=7.0
     - colorama >=0.4
+    - click >=7.0
     - cryptography
-    - graphene ==2.1.6
-    - jinja2 >=2.10.1
-    - markupsafe >=1.1
-    - metomi-isodatetime ==1!2.0.2
+    - graphene >=2.1,<3
+    - jinja2 ==2.11.0,<2.12
+    - metomi-isodatetime ==1!2.0.1
     - protobuf ==3.12.1
     - pyasn1
```

Comment:
How did this get in here?

Reply:
That was from one of the very first builds, when alpha1 was ready to be released. I remember adding this `pyasn1` and `cryptography` when building the first recipe.

But cannot say whether it is still necessary. I can drop it and run some tests tomorrow to confirm whether this, and `cryptography` are necessary, and add a note here... though I don't know if it was some subcommand that failed, or if it was running a workflow, ...

Reply:
Ok, not urgent, would be good to stick a comment in if it turns out to be necessary (I've never heard of pyasn1 before).

Reply:
Had to google to see what was pyasn1 too :smile: I only remember adding it :-( created #8 to track that later.